01/ht/gendos379/01/script.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Makes calculations according to given attributes.
+ * All attributes should be given in separated parenthesis, e.g.:
+ * calc(3)('+')(2)
+ * @param {Number} a - First number for calculation
+ * @param {String} b - Arithmetic operator, possible values: '+' / '-' / '/' / '*'
+ * @param {Number} c - Second number for calculation
+ * @return {Number} - Result of calculation
+ */
+function calc(a) {
+  return function operate(b) {
+    if (b === '+') {
+      return function sum(c) {

```

Comment:
Looks like if you create the single function which will analyze operation - it takes less code